migration-guide/redhat-to-ibm.adoc

```diff
@@ -0,0 +1,426 @@
+
+{MAVEN_SITE_URL_BASE}[Apache Maven] is a software project management and comprehension tool.
+Based on the concept of a project object model (POM), Maven can manage a project's build, reporting and documentation from a central piece of information.
+{PRODUCT_LONG} relies on Maven in order to build various types of deployable artifacts, such as decision service applications.
+For a quick start see the {MAVEN_SITE_URL_GETTING_STARTED}[Maven Quick Start Guide].
+
+=== Update Maven Settings
+
+The settings element in the *_settings.xml_* file contains data used to define values which configure Maven execution in various ways, like the *_pom.xml_*, but should not be bundled to any specific project, or distributed to an audience.
+These include values such as the local repository location, alternate remote repository servers, and authentication information.
+There are two locations where a settings.xml file may live:
+
+- **The Maven install**: *_${maven.home}/conf/settings.xml_*
+- **A user's install**: *_${user.home}/.m2/settings.xml_*
+
+[NOTE]
+====
+The former `settings.xml` are also called global settings, the latter `settings.xml` are referred to as user settings.
+If both files exists, their contents gets merged, with the user-specific `settings.xml` being dominant.
+====
+
+[TIP] 
+====
+If you need to create user-specific settings from scratch, it's easiest to copy the global settings from your Maven installation to your *_${user.home}/.m2_* directory.
+Maven's default *_settings.xml_* is a template with comments and examples so you can quickly tweak it to match your needs.
+_If you are looking to learn more about how to manage your various Maven settings, please visit the https://maven.apache.org/settings.html[Maven Settings Reference]._
+====
+
+{PRODUCT_LONG} v{VERSION_OLD} product dependencies are hosted by {RH_MAVEN_URL_BASE}[Red Hat's Maven repository], so you need to ensure that your `settings.xml` reflects that, as seen in the following example:
+
+include::../partials/settings-redhat-maven.adoc[]
+
+=== Update to use the Maven Bill of Materials (BOM)
+
+A bill of materials (BOM) is a list of the parts or components that are required to build a product.
+BOMs are ordinary `pom.xml` files — they contain no source code and their only purpose is to declare their bundled modules.
+
+.Always use the Maven Bill-of-Materials (BOM)
+[IMPORTANT]
+====
+It is highly recommended to use a bill of materials (BOM) to manage project depencencies, as it simplifies the dependency management by organizing a group of depdenencies under a single version. 
+====
+
+A bill of materails (BOM) can be added to an existing POM file by adding it to the **_dependencyManagement_** section as a dependency with a pom type:
+
+include::../partials/maven-bom-now.adoc[]
+
+When a bill of materials (BOM) is used it is no longer necessary to specify the depenency's version, as that comes from the BOM, so your depenencies end up looking like this:
```

Comment:
I'd also mention that using BOM won't bring any dependency to the project. it only helps solving version of project's declared dependencies and its transients

Reply:
I think it's worth to add the content about BOM not add dependencies. just copy from other file.